Reneiski_Homework1/Reneiski_Homework1/Lecture5/Zavod.cs

```diff
@@ -0,0 +1,29 @@
+﻿using WorkerNamespace;
+
+namespace Reneiski_Homework1.Lecture5
+{
+    public class Zavod
```

Comment:
Add method that returnes number of workers (should be done after adding of arrays of workers into Zavod)

Reply:
Fixed

Reply:
looks like still relevant